pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/activity/definition/question/PicklistQuestionDef.java

```diff
@@ -121,6 +122,14 @@ public Template getPicklistLabelTemplate() {
     }
 
     public List<PicklistOptionDef> getPicklistOptions() {
+        if (renderMode == PicklistRenderMode.REMOTE_AUTOCOMPLETE) {
+            return Collections.emptyList();
+        } else {
+            return picklistOptions;
+        }
+    }
+
+    public List<PicklistOptionDef> getPicklistOptionsIncludingRemoteAutoComplete() {
```

Comment:
Scenarios like elastic activity definition  export  and PATCH answers need REMOTE_AUTOCOMPLETE PL options included.